Integrations/integration-HashiCorp-Vault.yml

```diff
@@ -620,14 +621,17 @@ script:
         for engine in engines_to_fetch_from:
             if engine['type'] == 'KV':
                 if 'version' not in engine:
-                    return_error('Version not configured for KV engine, Re-configure the engine')
+                    return_error('Version not configured for KV engine, re-configure the engine')
                 if engine['version'] == '1':
                     credentials += get_kv1_secrets(engine['path'])
                 elif engine['version'] == '2':
                     credentials += get_kv2_secrets(engine['path'])
             elif engine['type'] == 'Cubbyhole':
                 credentials += get_ch_secrets(engine['path'])
 
+        if identifier:
```

Comment:
No way to query hashicorp for this specific record? 
Also need to make sure it returns only 1 value

Reply:
@orius123 There is a way, however, I have no way of knowing which engine I need when I have the identifier and if I query an engine that doesn't have that identifier I get a 404 error.
Users might configure more than 1 set of credentials, so I think an error in that case is the way to go.

Reply:
but if we fetch the wrong engine we will get no results also with this lambda, no?

Reply:
We fetch everything anyway, then filter according to the identifier